src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,14 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.card__review-label {
```

Comment:
In `normalize/reset` files you set base styles to tags themselves to reset in general. Styles related to card should in corresponding file